diff --git a/node_modules/lit-html/development/lit-html.js b/node_modules/lit-html/development/lit-html.js
index 8b732ae..a1962ee 100644
--- a/node_modules/lit-html/development/lit-html.js
+++ b/node_modules/lit-html/development/lit-html.js
@@ -9,6 +9,7 @@ const ENABLE_SHADYDOM_NOPATCH = true;
 const NODE_MODE = false;
 // Allows minifiers to rename references to globalThis
 const global = globalThis;
+const __moz_domParser = new DOMParser();
 /**
  * Useful for visualizing and logging insights into what the Lit template system is doing.
  *
@@ -657,9 +658,11 @@ class Template {
     // Overridden via `litHtmlPolyfillSupport` to provide platform support.
     /** @nocollapse */
     static createElement(html, _options) {
-        const el = d.createElement('template');
-        el.innerHTML = html;
-        return el;
+        const doc = __moz_domParser.parseFromString(
+            `<template>${html}</template>`,
+            'text/html'
+        );
+        return document.importNode(doc.querySelector('template'), true);
     }
 }
 function resolveDirective(part, value, parent = part, attributeIndex) {
